### PR TITLE
ADD PriceFilterWidget

### PR DESCRIPTION
**Pull request by cjohannsen**
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested by the author
- [x] Changes have been tested by the reviewer
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: resources/lang/de/Widget.properties [hunk 1]
```diff
@@ -861,6 +861,8 @@ paginationShowLastPageLabel = ""Zur letzten Seite"-Button anzeigen"
 attributesPropertiesCharacteristicsFilterLabel = "Filter für Attribute, Eigenschaften und Merkmale"
 ; Availability filter
 availabilityFilterLabel = "Filter für Verfügbarkeiten"
+; Price filter
+priceFilterLabel = "Filter für Preise"
```
Comment (fmutschler) [inline resources/lang/de/Widget.properties]:
```suggestion
priceFilterLabel = "Preisfilter"
```

File: resources/lang/en/Widget.properties [hunk 2]
```diff
@@ -860,6 +860,8 @@ paginationShowLastPageLabel = "Display "To last page" button"
 attributesPropertiesCharacteristicsFilterLabel = "Filter for attributes, properties and characteristics"
 ; Availability filter
 availabilityFilterLabel = "Filter for availabilities"
+; Price filter
+priceFilterLabel = "Filter for prices"
```
Comment (fmutschler) [inline resources/lang/en/Widget.properties]:
```suggestion
priceFilterLabel = "Price filter"
```

**Review by fmutschler — DISMISSED** [review]

**Review by stentrop — CHANGES_REQUESTED** [review]
geht aktuell nicht

**Comment by cjohannsen** [comment]
> geht aktuell nicht

done https://github.com/plentymarkets/plugin-ceres/pull/1670/commits/17f57765f7bb98487b17f5586b58cded59da083e

**Review by stentrop — APPROVED** [review]

**Pull request merged**